kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java

```diff
@@ -38,11 +40,6 @@
     @XStreamAlias("score")
     private ScoreWrapper scoreWrapper;
 
-    @XmlElement(name = "planning-problem")
-    @XStreamAlias("planning-problem")
-    @XmlJavaTypeAdapter(JaxbUnknownAdapter.class)
-    private Object planningProblem;
-
```

Comment:
@solutionjh 
The Kie Server API has changed in 7.0.x, please see https://github.com/kiegroup/kie-docs/blob/master/docs/optaplanner-wb-es-docs/src/main/asciidoc/Workbench/Quickstart/Quickstart-section.adoc#submit-solution to check how to submit a planning problem in a new way.

Register a solver:
PUT http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}

Submit a solution:
POST http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}/state/solving

Get the best solution:
GET http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}/bestsolution

Terminate the solver:
POST http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}/state/terminating-early

Dispose a solver:
DELETE http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}

Regards,
Matej